templates/docker.service.j2

```diff
@@ -20,7 +19,7 @@ LimitNPROC=infinity
 LimitCORE=infinity
 # Uncomment TasksMax if your systemd version supports it.
 # Only systemd 226 and above support this version.
-TasksMax=infinity
+#TasksMax=infinity
```

Comment:
Why this is changed?

Reply:
On the CentOS 7 Systemd has 219 version and this option doesn't work with version lower than 226 as you wrote in the comment. I decided to comment the option because without this it Docker works on a CentoOS and Ubuntu both.